workitem_test.go

```diff
@@ -44,9 +44,9 @@ func TestGetWorkItem(t *testing.T) {
 	repo := models.NewWorkItemRepository(ts)
 	controller := WorkitemController{ts: ts, wiRepository: repo}
 	payload := app.CreateWorkitemPayload{
-		Name: "foobar",
 		Type: "1",
```

Comment:
I think `Type: "1"` will need to become a real work item type name because work item types no longer have an ID field.